commands/debug/shell.go

```diff
@@ -17,12 +17,12 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func debugShellCmd(dockerCli command.Cli) *cobra.Command {
+func shellCmd(dockerCli command.Cli) *cobra.Command {
 	var options control.ControlOptions
 	var progressMode string
 
 	cmd := &cobra.Command{
-		Use:   "debug-shell",
+		Use:   "shell",
```

Comment:
IMO, we could just have `shell` be the top-level command, so `docker buildx debug` drops you directly into the shell.

Then we don't need a separate command for it, and it kind of acts as `debug` is your portal to all debugging stuff. The top-level `debug` command needs to do *something* anyways, and to me this makes the most sense.

Also, if we drop `shell`, it means that we could potentially have every command after `debug` can be run without the `debug` - `debug build` corresponds to `build`, `debug bake` corresponds to `bake`, etc.

Reply:
Thanks for the suggestion! Fixed the patch to launch the debugger directly by `buildx debug` command.